### PR TITLE
[FC] Update Romanian translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

File: base/applications/cmdutils/fc/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,41 +1,48 @@
+/*
+ * PROJECT:     ReactOS Command Prompt
+ * LICENSE:     LGPL-2.0-or-later (https://spdx.org/licenses/LGPL-2.0-or-later)
```
Comment (julenuri) [inline base/applications/cmdutils/fc/lang/ro-RO.rc:3]:
```suggestion
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
```

File: base/applications/cmdutils/fc/lang/ro-RO.rc
Comment (julenuri) [inline base/applications/cmdutils/fc/lang/ro-RO.rc:1]: Please, change the "[CMDUTILS]" from the title of the PR to "[FC]". Thanks.

**Review by julenuri — CHANGES_REQUESTED** [review]

File: base/applications/cmdutils/fc/lang/ro-RO.rc [hunk 2]
```diff
@@ -1,41 +1,48 @@
+/*
+ * PROJECT:     ReactOS Command Prompt
```
Comment (julenuri) [inline base/applications/cmdutils/fc/lang/ro-RO.rc:2]:
```suggestion
 * PROJECT:     ReactOS FC Command
```

**Review by julenuri — CHANGES_REQUESTED** [review]

File: base/applications/cmdutils/fc/lang/ro-RO.rc
Comment (AndreiMiloiu) [reply base/applications/cmdutils/fc/lang/ro-RO.rc:1]: Ok, I did it!

**Pull request merged**